commands/houndCommands.mjs

```diff
@@ -1,4 +1,4 @@
-import { sendMessageWithOptions } from '../utils/Utils.mjs';
+import { sendMessageWithOptions } from '../utils/Utils.mjs'; // eslint-disable-line import/extensions
```

Comment:
I am partial to putting this comment above the import statement and not next to it